app/distributions/api.py

```diff
@@ -0,0 +1,149 @@
+from ninja import Router
+from schemas import TranslationsSchema
+from utils.language import LanguageCode
+from utils.language import get_language
+from utils.language import get_translation
+
+from django.http import HttpRequest
+from django.shortcuts import get_object_or_404
+
+from .models import Attribution
+from .models import Dataset
+from .schemas import AttributionListSchema
+from .schemas import AttributionSchema
+from .schemas import DatasetListSchema
+from .schemas import DatasetSchema
+
+router = Router()
+
+
+def attribution_to_response(model: Attribution, lang: LanguageCode) -> AttributionSchema:
+    """
+    Transforms the given model using the given language into a response object.
+    """
+    response = AttributionSchema(
+        id=str(model.id),
+        name=get_translation(model, "name", lang),
+        name_translations=TranslationsSchema(
+            de=model.name_de,
+            fr=model.name_fr,
+            en=model.name_en,
+            it=model.name_it,
+            rm=model.name_rm,
+        ),
+        description=get_translation(model, "description", lang),
+        description_translations=TranslationsSchema(
+            de=model.description_de,
+            fr=model.description_fr,
+            en=model.description_en,
+            it=model.description_it,
+            rm=model.description_rm,
+        ),
+        provider_id=str(model.provider.id),
+    )
+    return response
+
+
+@router.get("attributions/{attribution_id}", response={200: AttributionSchema}, exclude_none=True)
```

Comment:
Doc says, `exclude_none=True` leads to exclusion of None fields. Does that mean that for instance `name_translations.it` would be omitted if empty? Does that even have an effect when we serialize manually (see comment below).
(Maybe I just don't understand Ninja well enough yet 😀)

Reply:
Ok I just saw from the tests that that's apparently what we want 😅